### ISSUE TITLE
Subject line doesn't show up in the window

### ISSUE THREAD

**Issue by ILoveSkilledCoders** [thundernest/quicktext#393]
Hi there,

It's hard to explain, so here are images.

While the subject line shows up when I'm looking at the email, here it's not showing up & that concerns me that it won't show up on the other person's side.

https://i.imgur.com/9YVkvvB.png

I should see the subject line when looking at all of my mail windows that are open in the list.

https://i.imgur.com/17XLDvn.png

Thanks

**Comment by jobisoft** [comment]
As soon as you actually enter a subject in the composed message, the title should update. If not, please try if it is fixed if the pre-release of v6:

Pre-release: https://github.com/jobisoft/quicktext/issues/439

**Comment by ILoveSkilledCoders** [comment]
Hi,

Does that work on vs 102.15.1 b/c I am not upgrading to the newest version?

Thanks

**Comment by jobisoft** [comment]
I have never been able to reproduced your issue. For me, the title of the compose window is always showing the current subject. I also do not see how this is related to Quicktext.

**Comment by ILoveSkilledCoders** [comment]
It shows it in the email when you are looking at it, but if I bring up all of the TB windows that are open, it doesn't show up. It says No subject even though there's a subject in the email because I have a subject in that email in QT. And this is all done thru QuickText, so yes, it is only about QT.

Here's another SS - https://i.imgur.com/6YSsfOq.png

Do you understand now?

**Comment by jobisoft** [comment]
No. I do not understand, sorry. Please disable quicktext and check if the issue is resolved.
